### PR TITLE
Improve_logging - Make the logging consistent

### PR DESCRIPTION
**Pull request by svedbg**
Minor improvements in the logging to be more consistent

### CODE REVIEW

**Comment by liadgabriel** [comment]
Hi svedbg,
thank you for that, I will review it soon

File: src/copy-object-multipart.js [hunk 1]
```diff
@@ -113,14 +113,14 @@ function abortMultipartCopy(destination_bucket, copied_object_name, upload_id, r
         })
         .then((parts_list) => {
             if (parts_list.Parts.length > 0) {
-                logger.error({ msg: 'abort multipart copy failed, copy parts were not removed', context: request_context, parts_list: parts_list });
-
                 const err = new Error('Abort procedure passed but copy parts were not removed');
                 err.details = parts_list;
 
+                logger.error({ msg: `abort multipart copy failed, copy parts were not removed: ${JSON.stringify(parts_list)}`, context: request_context, error: err });
```
Comment (LiadBerko) [inline src/copy-object-multipart.js]: since parts_list is in err.details it is printed to error log twice. Please change that

**Comment by LiadBerko** [comment]
Dear svedbg,
Note that the pipeline is failing.
Please merge with new master and retry, thanks!

File: src/copy-object-multipart.js (hunk 1 above)
Comment (svedbg) [reply src/copy-object-multipart.js]: Commit https://github.com/Zooz/aws-s3-multipart-copy/pull/15/commits/eb92aa9191b23220f4a5c06821e7fb6d6b5d4fe4 fixes the twice log. Also branch was rebased from master (sorry for the force push). The pipeline now is passing.

**Pull request merged**